test/e2e/pkg/testnet.go

```diff
@@ -84,36 +81,31 @@ type Testnet struct {
 
 // Node represents a CometBFT node in a testnet.
 type Node struct {
-	Name                  string
-	Version               string
-	Testnet               *Testnet
-	Mode                  Mode
-	PrivvalKey            crypto.PrivKey
-	NodeKey               crypto.PrivKey
-	IP                    net.IP
-	ProxyPort             uint32
-	StartAt               int64
-	FastSync              string
-	StateSync             bool
-	Mempool               string
-	Database              string
-	ABCIProtocol          Protocol
-	PrivvalProtocol       Protocol
-	PersistInterval       uint64
-	SnapshotInterval      uint64
-	RetainBlocks          uint64
-	Seeds                 []*Node
-	PersistentPeers       []*Node
-	Perturbations         []Perturbation
-	Misbehaviors          map[int64]string
-	SendNoLoad            bool
-	Prometheus            bool
-	PrometheusProxyPort   uint32
-	TracePushConfig       string
-	TracePullAddress      string
-	PyroscopeURL          string
-	PyroscopeTrace        bool
-	PyroscopeProfileTypes []string
+	Name                string
+	Version             string
+	Testnet             *Testnet
```

Comment:
if we do every need tracing or pyroscope to quickly debug things, we can simply change the defaults or elswhere in the pipeline. I don't think we need to explicity support these things in the e2e test